### DISCUSSION TITLE
2024.09.26 - Community Meeting #10061

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#10332] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfil this, we would like to held Bimonthly Community Meeting where we can share main updates and also have an open discussion with you.

---

**💥 Eighteenth Community Meeting will take place on Thursday 26/09/2024 at 4 pm GMT+2 time zone. (1 hour length).**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you already participated on any Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you join us again or not.

---

### Agenda will be specified below in the comments:  (feel free to add any topic or idea)

1) Development updates & plans

2) Community Topics

3) Open Discussion

---

See you on the Meeting!

**Comment by williamsokol** [comment]
Do we still do these meetings? It seems like this last one posted here was almost a year ago

**Comment by SuGlider** [comment]
@Parsaabasi @rftafas - Is there any plan for having these meeting again?

**Reply by rftafas** [reply]
No plans.

1. Due to the very wide time zones of our audience, it is impossible to get a time that fits all.
2. Discord is showing itself as a great outlet to let community reach our team for questions and doubts.
3. For more 'forum like' behavior, there is GH discussions.
4. Discussions and Discord have been a far better outlet for announcements (partially due to 1)
5. Issues are confirmed to be the best place to report bugs, make feature requests or even to manage our own tasks.
6. There is Espressif DEVCON, which always hold great content for Arduino.
7. Same with the "Developer's Portal", great content there.

So, we are taking this time and energy for the above. IF there is commotion from community, we can return with this.
